euphonic/qpoint_frequencies.py

```diff
@@ -107,6 +110,12 @@ def calculate_dos(self, dos_bins: Quantity,
             Scalar float Quantity in energy units. Sets a lower limit on
             the mode widths, as mode widths of zero will result in
             infinitely sharp peaks
+        adaptive_method
```

Comment:
I wonder if we could name the existing method 'reference'; it would be nice to make the new method the default, once it is robust enough!

Reply:
I was initially going to suggest 'exact', but that might mislead people away from the fact that adaptive broadening is itself an approximation to speed up q-point convergence.

Reply:
Good point!